Update setup.sh

xstefen opened the pull request:
Remove apt installation of file-roller which attempts to bring in a ton of X11/gtk dependencies which are absolutely useless on a headless system. File-roller is not required to use dumpyara in any case.

erfanoabdi commented:
./dumpyara.sh: line 125: [: Erfan: binary operator expected

ShivamKumarJha commented:
weird as i didnt face it

ShivamKumarJha commented:
btw if we are open to telegram channel, this thing works https://github.com/ShivamKumarJha/dumpyara/commit/1d1d287cf52597509289c624cc91f1d222688269

Commit:
Update setup.sh

Remove apt installation of file-roller which attempts to bring in a ton of X11/gtk dependencies which are absolutely useless on a headless system. File-roller is not required to use dumpyara in any case.
